apps/prairielearn/src/lib/load.ts

```diff
@@ -112,7 +124,7 @@ class LoadEstimator {
     debug(`LoadEstimator._warnOldJobs(): jobType = ${this.jobType}`);
     if (!this.warnOnOldJobs) return;
     const nowMS = Date.now();
-    _.forEach(this.currentJobs, (/** @type {object} */ info, id) => {
+    _.forEach(this.currentJobs, (info: JobLoad, id) => {
```

Comment:
Type inference should be sufficient here.

```suggestion
    _.forEach(this.currentJobs, (info, id) => {
```